documentation/architecture-decision-records/ADR0003-definition_of_done.md

```diff
@@ -0,0 +1,63 @@
+# Definition of Done
+## ADR-0003
+
+* Status: Proposed
+* Deciders:
+  * Articles Team
+  * WS Team
+* Date: 2019-08-08
+
+## Context
+
+We require a definitive description of when a particular piece of work in Psammead can be considered done.
+
+### Decision Drivers
+
+* An accessibility swarm can take time to organise. If we block merging into Psammead for time to arrange this, it could slow down development.
+* We want to test components as early as possible - before they've been merged and integrated into the application.
+* We want to ensure we have a balance that keeps high standards yet allows fast development.
+
+## When working on PRs that are not yet ready for production use:
+
+Use an alpha version - is where `package.json` / `package-lock.json` version has format `0.0.0.alpha.0` & `alpha tag`.
+
```

Comment:
I would also mention that we should add the following text:
```
# ⛔️ This is an alpha component ⛔️

This component is currently tagged as alpha and is not suitable for production use. Following the passing of an accessibility review this component will be marked as ready for production and the alpha tag removed.
```
at the top of the README.